rotkehlchen/tests/api/test_data_import.py

```diff
@@ -611,3 +612,21 @@ def test_bitcoin_tax_import(rotkehlchen_api_server):
     )
     assert assert_proper_response_with_result(response) is True
     assert_bitcoin_tax_trades_import_results(rotki, filepath.name)
+
+
+def test_bitstamp_import(rotkehlchen_api_server):
+    """Test that data import works for Bitstamp transaction csv files"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    dir_path = Path(__file__).resolve().parent.parent
+
+    # First test a trades type csv import
+    filepath = dir_path / 'data' / 'bitstamp.csv'
+    json_data = {'source': 'bitstamp', 'file': str(filepath)}
```

Comment:
please also update the api documentation under `docs/api.rst` and the user guide in the same folder